torch/csrc/jit/codegen/cuda/transform_iter.cpp

```diff
@@ -130,13 +131,63 @@ void ReplayTransformations::handle(Merge* m) {
   id_map_[m->out()] = out;
 }
 
+void ReplayTransformations::handle(Swizzle2D* swizzle_2d) {
+  // Grab our input to the split node
+  auto id_in_x = swizzle_2d->inX();
```

Comment:
Yes makes sense. A bit of detail though, `lhs`, and `inX` makes sense for `Binary` and `Swizzle`, but I'm uncertain what we should call the Ternary inputs following the same pattern.

Reply:
I don't think it matters much and we can come back to it since it's a simple replace. Anything consistent should be fine, it's mostly about predictability when coding for developer efficiency.